src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/MapView/DisplayLayerViewState/DisplayLayerViewState.xaml

```diff
@@ -15,19 +15,9 @@
             <StackPanel>
```

Comment:
Make layer information panel a little more compact, center and bold the title.
<Border
            Background="White" BorderBrush="Black" BorderThickness="1"
            HorizontalAlignment="Right" VerticalAlignment="Top"
            Margin="30" Padding="20" Width="**250**">
            <StackPanel>
                <TextBlock Text="Layer Display Status" **FontWeight="Bold" 
                      HorizontalAlignment="Center"**/>
                <Label x:Name="StatusLabel_TiledLayer" Content=""/>
                <Label x:Name="StatusLabel_ImageLayer" Content=""/>
                <Label x:Name="StatusLabel_FeatureLayer" Content=""/>
            </StackPanel>
        </Border>